src/Bicep.Core/Analyzers/Linter/LinterAnalyzer.cs

```diff
@@ -0,0 +1,116 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using Azure.Deployments.Core.Extensions;
+using Bicep.Core.Analyzers.Interfaces;
+using Bicep.Core.Configuration;
+using Bicep.Core.Diagnostics;
+using Bicep.Core.Parsing;
+using Bicep.Core.Semantics;
+using System;
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using System.Diagnostics;
+using System.Linq;
+using System.Reflection;
+
+namespace Bicep.Core.Analyzers.Linter
+{
+    internal class LinterAnalyzer : IBicepAnalyzer
+    {
+        public const string SettingsRoot = "analyzers";
+        public const string AnalyzerName = "bicepcorelinter";
+        public static string LinterEnabledSetting => $"{SettingsRoot}:{AnalyzerName}:enabled";
+        public static string LinterVerboseSetting => $"{SettingsRoot}:{AnalyzerName}:verbose";
```

Comment:
>LinterVerboseSetting [](http://example.com/codeflow?start=29&length=20)

Is there a way we could avoid mixing up config file logic with the rest of the analyzer? Perhaps if we had a strongly typed model for the config file that the analyzer could interact with directly?